src/ProjectTemplates/Web.ProjectTemplates/EmptyWeb-CSharp.csproj.in

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>${DefaultNetCoreTargetFramework}</TargetFramework>
     <NoDefaultLaunchSettingsFile Condition="'$(ExcludeLaunchSettings)' == 'True'">True</NoDefaultLaunchSettingsFile>
     <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">Company.WebApplication1</RootNamespace>
+    <LangVersion>preview</LangVersion>
```

Comment:
I *think* we'll be able to remove this when we get a new SDK.

cc @jaredpar

Reply:
This will go away in .NET 6 P7.

Reply:
@jaredpar can you help me understand this better as the docs seem to say preview frameworks will always default to their matching preview language: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version